api-reference/openapi.yaml

```diff
@@ -2735,3 +2735,159 @@ components:
+    dataCaptureFormSubmissionVersionModel:
+      type: object
+      properties:
+        type:
+          type: string
+          example: versions
+          enum:
+            - versions
+          description: The resource type
+        id:
+          type: string
+          format: uuid
+          example: 7813209e-db7b-4d3d-82a9-73c3c1b10090
+          description: The ID of the form submission version
+        attributes:
+          type: object
+          properties:
+            status:
+              type: string
+              description: The status of the form submission
+              example: complete
+            content: 
+              type: object
+              description: The content of the submission
+        meta:
+          type: object
+          properties:
+            created_at:
+              type: string
+              example: 2024-06-13T07:15:23Z
+              description: "**Sortable**. The time the operation was created ([RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339))"
+        relationships:
+          type: object
+          properties:
+            form_version:
+              type: object
+              description: The id of form version this submission version relates to
+              properties:
+                data:
+                  type: array
+                  items:
+                    $ref: "#/components/schemas/dataCaptureFormVersionRelationship"
+            author:
+              type: object
+              description: The id of user this submission version relates to
+              properties:
+                data:
+                  type: array
+                  items:
+                    $ref: "#/components/schemas/userRelationships"
+        links:
+          type: object
+          properties:
+            self:
+              type: string
+              example: https://api.ctrl-hub.com/v3/orgs/7e0c875a-fd47-401d-bd25-4278ed58d721/submissions/eb2b4aa7-048d-470b-b504-a3eb3bf45e29/relationships/versions/7813209e-db7b-4d3d-82a9-73c3c1b10090
+    dataCaptureFormSubmissionModel:
+      type: object
+      properties:
+        type:
+          type: string
+          example: submissions
+          enum:
+            - submissions
+          description: The resource type
+        id:
+          type: string
+          format: uuid
+          example: eb2b4aa7-048d-470b-b504-a3eb3bf45e29
+          description: The ID of the form submission
+        attributes:
+          type: object
+          properties:
+            reference:
+              type: string
+              description: The reference of the form submission
+              example: 1234A567BCD
+            status:
+              type: string
+              description: The status of the form submission
+              example: complete
+            content:
+              type: object
+              description: The content of the form submission
```

Comment:
Will this pull the content from the latest submission version? Or is it redundant here and instead we rely on API client navigating to `/v3/orgs/{org}/data-capture/submissions/{id}/relationships/versions/latest`?

My preference is for the latter - we can always add an optional `?include=latest` or something similar to pull back the content, but doing so by default will increase the payload on the wire here.